Demo/Demo/UseCases/GetAccessToken.swift

```diff
@@ -0,0 +1,8 @@
+import UIKit
+
+final class GetAccessToken {
+
+    func execute() async -> String? {
```

Comment:
Why are we wrapping `getAccessToken()` in another method?

Reply:
The idea is that the viewModels interact with useCases, and inside the usecases are the calls to the data layer. So even if its a simple request, it should be inside a useCase.

Reply:
I don't think we need an additional layer wrapping the data layer. In [MVVM](https://www.raywenderlich.com/6733535-ios-mvvm-tutorial-refactoring-from-mvc), the ViewModel is meant to be the bridge b/w the View and Data layers. I think the UseCase layer adds extra complexity to the Demo.